src/functions.ts

```diff
@@ -905,7 +905,9 @@ export async function collectData(
   return Logger.traceSdkMethod(async () => {
     try {
       const response = await StripeTerminalSdk.collectData(params);
-      return response;
+      return {
+        collectedData: response
```

Comment:
CI complain about the missing trailing commas style here.
> /bitrise/src/src/functions.ts
  909:32  error    Insert `,`              prettier/prettier

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas